glass/src/lib/native/cpp/support/ExtraGuiWidgets.cpp

```diff
@@ -217,4 +222,68 @@ bool HamburgerButton(const ImGuiID id, const ImVec2 position) {
   return pressed;
 }
 
+struct InputExprState {
+  char* inputBuffer = nullptr;
+  bool wasActive = false;
+};
+
+static const int kBufferSize = 256;
+
+static std::unordered_map<int, InputExprState> exprStates;
```

Comment:
We tend to use `wpi::DenseMap` for these use cases, rather than `std::unordered_map`, but I'm not hard-over on that.